### PR TITLE
updating workflows - master

### PR DESCRIPTION
**Pull request by ATorrise**
Action | Description
-- | --
#1 | add [stale bot](https://github.com/zowe/zowe-cli/blob/master/.github/workflows/stale.yml)	
#2 | add [auto-comment](https://github.com/zowe/zowe-cli/blob/master/.github/workflows/auto-comment.yml) bot [on labels](https://github.com/zowe/zowe-cli/blob/master/.github/label-actions.yml)										
#3 | add issue template ([Bug](https://github.com/zowe/zowe-cli/blob/master/.github/ISSUE_TEMPLATE/bug_report.md), [Feature](https://github.com/zowe/zowe-cli/blob/master/.github/ISSUE_TEMPLATE/feature_request.md))										
#4 | add [PR Template ](https://docs.google.com/document/u/0/d/1BvA8zNQs6QP1_gHJw7XsbEGNTTCH-v0SjM4kwMym3Cw/edit)   										
#5 | add [question redirect](https://github.com/zowe/zowe-cli/blob/master/.github/ISSUE_TEMPLATE/config.yml)										
#6 | [update all workflows to remove deprecation warnings](https://github.com/zowe/zowe-cli-web-help-generator/pull/35/files), [octorelease updates](https://github.com/zowe/zowe-cli/pull/1712)										
#7 | remove Node 14 from workflow matrix

### CODE REVIEW

File: .github/workflows/changelog.yml
Comment (zFernand0) [inline .github/workflows/changelog.yml:1]: Please update the `awharn/check_changelog_action@v0.0.2` to `v1` too 😉

File: .github/workflows/codeql.yml [hunk 1]
```diff
@@ -30,10 +30,10 @@ jobs:
 
     steps:
     - name: Checkout repository
-      uses: actions/checkout@v2
+      uses: actions/checkout@v3
 
     - name: Initialize CodeQL
-      uses: github/codeql-action/init@v1
```
Comment (zFernand0) [inline .github/workflows/codeql.yml:36]: thanks for updating the `init`. can we do the same for the `analyze` step?

File: .github/workflows/zowe-cli-plugin.yml [hunk 2]
```diff
@@ -11,7 +11,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        node-version: [14.x, 16.x, 18.x]
```
Comment (zFernand0) [inline .github/workflows/zowe-cli-plugin.yml:14]: now that npm8 is bundled with the remaining version of node, I believe we can remove the "update npm" step

File: .github/workflows/zowe-cli-plugin.yml [hunk 3]
```diff
@@ -59,7 +59,7 @@ jobs:
 
     - name: Archive Results
       if: ${{ always() && steps.build.outcome == 'success' }}
-      uses: actions/upload-artifact@v2
+      uses: actions/upload-artifact@v3
       with:
         name: ${{ matrix.os }}-${{ matrix.node-version }}-results
         path: |
```
Comment (zFernand0) [inline .github/workflows/zowe-cli-plugin.yml:65]: could you also update the codecov-action to v3 here?

**Review by zFernand0 — CHANGES_REQUESTED** [review]

File: .github/workflows/zowe-cli-plugin.yml (hunk 3 above)
Comment (t1m0thyj) [reply .github/workflows/zowe-cli-plugin.yml:62]: This still needs to be updated

**Review by t1m0thyj — CHANGES_REQUESTED** [review]

**Review by zFernand0 — APPROVED** [review]

**Pull request merged**